From: Mieczyslaw Nalewaj <namiltd@yahoo.com>
Date: Mon, 18 Nov 2024 21:28:05 +0100
Subject: mips: ralink: add missing include

Missing headers causes an error on kernel 6.12:
arch/mips/ralink/irq.c:86:5: error: no previous prototype for 'get_c0_perfcount_int' [-Werror=missing-prototypes]
arch/mips/ralink/irq.c:92:14: error: no previous prototype for 'get_c0_compare_int' [-Werror=missing-prototypes]
cc1: all warnings being treated as errors

Signed-off-by: Mieczyslaw Nalewaj <namiltd@yahoo.com>

---
 arch/mips/ralink/irq.c | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/mips/ralink/irq.c
+++ b/arch/mips/ralink/irq.c
@@ -15,6 +15,7 @@
 
 #include <linux/init.h>
 #include <linux/irq.h>
+#include <asm/time.h>
 
 #include "common.h"
 
